stdlib/public/Concurrency/YieldingContinuation.swift

```diff
@@ -0,0 +1,201 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2020-2021 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import Swift
+
+@_fixed_layout
+@usableFromInline
+internal final class _YieldingContinuationStorage: UnsafeSendable {
+  @usableFromInline
+  var continuation: Builtin.RawUnsafeContinuation?
+}
+
+@available(macOS 9999, iOS 9999, watchOS 9999, tvOS 9999, *)
+public struct YieldingContinuation<Element, Failure: Error>: Sendable {
+  @usableFromInline
+  let storage = _YieldingContinuationStorage()
```

Comment:
Why is this `@usableFromInline`?  The existence of this shouldn't be exposed if you want to reserve the ability to meaningfully change the representation in the future.  If you don't want to reserve that, the struct should be frozen.

Reply:
that means that I will need to have the inject/extract non inline then - which I think might be worth it here for resilience.

Reply:
Yeah, being inlinable does nothing for them, because they aren't actually used from inlinable code.  If you want to be resilient, just be resilient all the way down.